bin/oref0-online.sh

```diff
@@ -2,13 +2,17 @@
 MAC=$1
 echo -n "At $(date) my IP is: "
 if ! curl -m 15 icanhazip.com; then
-    echo -n "Error, cycling wlan0 "
-    sudo ifdown wlan0
-    sudo ifup wlan0
-    echo -n "and getting new wlan0 IP"
-    ps aux | grep -v grep | grep -q "dhclient wlan0" && sudo killall dhclient
-    sudo dhclient wlan0 -r
-    sudo dhclient wlan0
+    echo -n "Error, cycling networking "
+   # sudo ifdown wlan0
+  #  sudo ifup wlan0
+  #  echo -n "and getting new wlan0 IP"
```

Comment:
Does `/etc/init.d/networking stop` kill dhclient?  If not, it might be worthwhile keeping this part (and put it after the networking restart) in case we need still to kick DHCP.